### ISSUE TITLE
GPL License

### ISSUE THREAD

**Issue by bernhardttom** [telefonicaid/perseo-core#131]
Remove this, it is wrong:

_Software that merely uses or aggregates (i.e. links to) an otherwise unmodified version of existing software is not considered a derivative work, and therefore it does not need to be released as under the same license, or even released as open source._

**Comment by bernhardttom** [comment]
You are NOT ENTITLED to give any legal advice.

**Comment by fgalan** [comment]
I think it was @jason-fox who added that part to the README.md

Jason, any opinion or feedback regarding this, please?

**Comment by jason-fox** [comment]
The addition of a clarification statement to FIWARE GEs was agreed at TSC level and is **mandatory**.  I disagree with @bernhardttom opinion that this statement constitutes legal advice, as he is cherry picking only part of the statement, specifically missing the part beforehand:

> the owners of this software... wish to make a clarifying public statement as follows

States that this is the _opinion of the owner_ - followed by the rest of the quote.

Interpretation of GPL vs derivative work is murky, and this is not an attempt at giving legal advice.
It is merely stating that if you use this software and for whatever reason the narrowest interpretation of GPL rights is tested in a court of law (which one by the way?) then since the narrow interpretation isn't the opinion of the owner of this software, you won't get sued by them.

In other words no bait and switch.

**Comment by bernhardttom** [comment]
You have no right to interpret or make advice about the GPL license publicly. 

You can instead perhaps write this on a blog or other private statement of some person, whoever TSC level and FIRMWARE GE may be.  Or provide author contact information for FIRMWARE GE TSC level.
